### DISCUSSION TITLE
[dns-packet]

### DISCUSSION THREAD

**Discussion by demosjarco** [DefinitelyTyped/DefinitelyTyped#67884] [Issues with a @types package]
In [mafintosh/dns-packet/blob/master/index.js#L1655](https://github.com/mafintosh/dns-packet/blob/master/index.js#L1655) `DNSSEC_OK` is being exported, but this package doesn't:
`Module '"dns-packet"' has no exported member 'DNSSEC_OK'.ts(2305)`

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
Added in PR: #68270
